libs/libcommon/src/libcommon/queue.py

```diff
@@ -238,6 +246,34 @@ def flat_info(self) -> FlatJobInfo:
         )
 
 
+DEFAULT_INCREASE_AMOUNT = 1
+DEFAULT_DECREASE_AMOUNT = -1
+
+
+class JobTotalMetricDocument(Document):
+    """Jobs total metric in mongoDB database, used to compute prometheus metrics.
+
+    Args:
+        queue (`str`): queue name
```

Comment:
maybe job_type instead of queue, as it's how we call it in the rest of the code

Reply:
Done in https://github.com/huggingface/datasets-server/pull/1684/commits/721689f02a3fd58e117a8a4bf0033a5bd883d311